front/app/components/admin/PostManager/components/FilterSidebar/statuses/StatusButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Button, colors, fontSizes } from '@citizenlab/cl2-component-library';
+
+interface Props {
+  children: React.ReactNode;
+  onClick: () => void;
+  disabled?: boolean;
+  active: boolean;
+}
+
+const StatusButton = ({
+  children,
+  onClick,
+  disabled = false,
+  active,
+}: Props) => {
```

Comment:
Would ideally have used FilterRadioButton (radio inputs) here too, but was too much work implementing the disabled state for the label so kept what we have for now.